flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/cli/parser/SqlCommandParser.java

```diff
@@ -16,17 +16,18 @@
  * limitations under the License.
  */
 
-package org.apache.flink.table.client.cli;
+package org.apache.flink.table.client.cli.parser;
 
 import org.apache.flink.annotation.Internal;
+import org.apache.flink.sql.parser.impl.FlinkSqlParserImplConstants;
 import org.apache.flink.table.client.gateway.SqlExecutionException;
 import org.apache.flink.table.operations.Operation;
 
 import java.util.Optional;
 
-/** SqlClient command parser. */
+/** SqlClient command parser. Extends FlinkSqlParserImplConstants to use defined token kinds. */
 @Internal
-interface SqlCommandParser {
+public interface SqlCommandParser extends FlinkSqlParserImplConstants {
```

Comment:
Why this interface extends `FlinkSqlParserImplConstants`? I think ClientParser extends this is enough.

Reply:
Removed.